cl/users/models.py

```diff
@@ -141,6 +141,11 @@ class UserProfile(models.Model):
         help_text="Sort dockets in descending order by default",
         default=False,
     )
+    neon_account_id = models.CharField(
+        help_text="Unique identifier assigned by Neon CRM to a customer record",
+        default="",
```

Comment:
One more here, I think.

Reply:
You are correct. We don't need to provide a default value when setting the 'blank' attribute to True.